app/uploaders/file_upload.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+class FileUpload
+  attr_accessor :options
+  class << self
+    def uploader(uploader = nil)
```

Comment:
I bounce back and forth on how to do class config methods, but you could do:

```ruby
class FileUpload
  class_attribute :uploader
  class_attribute :workflow

# then...
class MyFileUpload < FileUpload
  self.uploader = EVSS::Uploader
  self.workflow = EVSS::Workflow
```
removes the need for singleton class writers and wrapping self.class.thing in an instance accessor method but not as slick as a DSL 😀